src/main/java/com/softserve/kickscooter/kickscootervehicle/management/controller/KafkaController.java

```diff
@@ -16,36 +17,48 @@
 @Slf4j
 public class KafkaController {
 
-    private static final String STATUS_TOPIC = "all-scooter-data-topic";
+    private static final String IN_USE_TOPIC = "in-use-scooter-data";
+    private static final String FREE_TOPIC = "free-scooter-data";
     private static final String RAW_DATA = "raw-data";
 
     private final KafkaTemplate<String, ScooterStatusDto> template;
 
     private final StatusService statusService;
 
-    //todo: send to active scooter
-    //todo: send to free scooter topic
-    public void sendStatusDataToTopic(ScooterStatusDto dto){
-        log.info("Send data to topic '{}': {} ", STATUS_TOPIC, dto);
-        template.send(STATUS_TOPIC, dto);
+    public void sendInUseStatusDataToTopic(ScooterStatusDto dto){
+        log.info("Send data to topic '{}': {} ", IN_USE_TOPIC, dto);
+        template.send(IN_USE_TOPIC, dto);
+    }
+
+    public void sendFreeStatusDataToTopic(ScooterStatusDto dto){
+        log.info("Send data to topic '{}': {} ", FREE_TOPIC, dto);
+        template.send(FREE_TOPIC, dto);
     }
 
     @KafkaListener(topics = RAW_DATA, groupId = "${spring.kafka.consumer.group-id}")
     public void listen(ScooterRawDataDto rawDto) {
         log.info("Received from '{}' : {}", RAW_DATA, rawDto);
-        ScooterStatus status = statusService.getCurrentStatus(rawDto.getId());
-        var statusDto = new ScooterStatusDto();
-        statusDto.setId(rawDto.getId());
-        statusDto.setGpsPoint(new Point(rawDto.getLatitude(),rawDto.getLongitude()));
-        statusDto.setStatus(status);
-        statusDto.setBattery(rawDto.getBattery());
-
-        statusService.saveActualStatusData(rawDto.getId(),
-                rawDto.getLatitude(),
-                rawDto.getLongitude(),
-                rawDto.getBattery());
-
-        sendStatusDataToTopic(statusDto);
+        try {
+            ScooterStatus status = statusService.getCurrentStatus(rawDto.getId());
+            var statusDto = new ScooterStatusDto();
+            statusDto.setId(rawDto.getId());
+            statusDto.setGpsPoint(new Point(rawDto.getLatitude(), rawDto.getLongitude()));
+            statusDto.setStatus(status);
+            statusDto.setBattery(rawDto.getBattery());
+
+            statusService.saveActualStatusData(rawDto.getId(),
+                    rawDto.getLatitude(),
+                    rawDto.getLongitude(),
+                    rawDto.getBattery());
+
+            if(statusDto.getStatus() == ScooterStatus.IN_USE) {
+                sendInUseStatusDataToTopic(statusDto);
+            } else if (statusDto.getStatus() == ScooterStatus.FREE){
+                sendFreeStatusDataToTopic(statusDto);
+            }
+        } catch (ScooterIsDecommisionedException e){
+            log.error("Received from DECOMMISSIONED scooter '{}' : {}", RAW_DATA, rawDto);
```

Comment:
In what purpose you use this syntax  '{}' : {}. I mean, why second curly brackets  has no quotation mark?